explainaboard/loaders/ranking.py

```diff
@@ -0,0 +1,46 @@
+"""Loaders for the ranking task."""
+
+from __future__ import annotations
+
+from explainaboard import TaskType
+from explainaboard.constants import FileType
+from explainaboard.loaders.file_loader import (
+    DatalabFileLoader,
+    FileLoader,
+    FileLoaderField,
+    JSONFileLoader,
+    TextFileLoader,
+)
+from explainaboard.loaders.loader import Loader
+from explainaboard.loaders.loader_registry import register_loader
+
+
+@register_loader(TaskType.argument_pair_identification)
+@register_loader(TaskType.ranking)
+class RankingLoader(Loader):
+    """Loaders for the ranking tasks, such as argument pair identification."""
+
+    @classmethod
+    def default_dataset_file_loaders(cls) -> dict[FileType, FileLoader]:
+        """See Loader.default_dataset_file_type."""
+        target_field_names = ["context", "utterance", "true_label"]
```

Comment:
Are these `target_field_names` applicable to other ranking tasks? For example, what about the most typical ranking task of information retrieval?

There I feel that in this case we have only `query` and `true_label`?

Maybe to future-proof the naming, it would be more appropriate to name it `ranking_with_context` and `utterance` should be replaced with `query`?

Also apologies for all the iteration on this, but if we set this now and change it later it might affect the validity of the saved DB entries, so we probably want to try to get it right the first time.